arch/x86/core/crt0.S

```diff
@@ -283,6 +284,25 @@ bssWords:
 	lgdt	%ds:_gdt
 #endif
 
+#ifdef CONFIG_X86_MMU
+
+	/* load the page directory address into the registers*/
+	movl $__mmu_tables_start, %eax
+	movl %eax,%cr3
+
+	/*Set CR4.PAE = 0 (5th bit in CR4*/
+	movl %cr4, %eax
+	andl $~0x10, %eax
+	movl %eax, %cr4
+
+	/* set CR0.PG bit (31st bit in CR0)*/
+	movl %cr0,%eax
+	orl $0x80010000, %eax
```

Comment:
are there any defined constants in kernel_arch_data.h we could use or add instead of having a magic number directly here?